app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+
+// @ts-ignore
+import { Button } from "neetoui";
+// @ts-ignore
+import { Container, Header } from "neetoui/layouts";
+
+import Menu from "./Menu";
+import Table from "./Table";
+
+const Contacts = () => {
+  // TODO: Add add function to fetch contact details from server and render PageLoader component till completion.
+
+  const [showMenu, setShowMenu] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  return (
+    <>
+      <Menu showMenu={showMenu} />
+      <Container>
+        <Header
+          menuBarToggle={() => setShowMenu(!showMenu)}
```

Comment:
https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update